README.md

```diff
@@ -15,6 +15,11 @@ You can reach the maintainers of this project at:
 
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
 
+## How it works:
+
+Currently the driver makes use of buildah to download the container image if it is not already available, launch a new instance of it named after the volumeHandle, and mount it.
+
+In the future, integration with CRI would be desirable so the driver could ask via CRI that the Container Runtime perform these activities in a generic way.
 
 ## Usage:
 
```

Comment:
Maybe add a note somewhere here about the min K8s version (master or still under dev)

Reply:
+1